crates/sui-indexer/migrations/pg/2024-09-25-211901_drop_network_total_transactions/up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE checkpoints DROP COLUMN network_total_transactions;
+UPDATE checkpoints SET max_tx_sequence_number = max_tx_sequence_number + 1 WHERE max_tx_sequence_number IS NOT NULL;
```

Comment:
this might take mins to finish, I can split this change into steps
- adding a new column like `new_max_tx_sequence_number` and populate both
- backfill `new_max_tx_sequence_number`
- remove `max_tx_sequence_number` and rename `new_max_tx_sequence_number` to `max_tx_sequence_number`

it's a bit wasteful .. but I am okay to do that if needed. cc @lxfind

Reply:
How about if you dropped `max_tx_sequence_number` and renamed `network_total_transactions`? (modulo the conversation on what the name should be, below)

Reply:
great idea, thanks!